src/util.ts

```diff
@@ -31,3 +31,6 @@ export function range(count: number): number[] {
   }
   return ret
 }
+
+export const DEFAULT_MAX_FRAME_SIZE = 1048576
+export const DEFAULT_UNLIMITED_FRAME_SIZE = 0
```

Comment:
same here DEFAULT_UNLIMITED_MAX_FRAME or DEFAULT_UNLIMITED_MAX_FRAME_SIZE if the size part is relevant